src/sql/base/browser/ui/table/plugins/checkboxSelectColumn.plugin.ts

```diff
@@ -243,32 +243,37 @@ export class CheckboxSelectColumn<T extends Slick.SlickData> implements Slick.Pl
 		}
 
 		return this._selectedRowsLookup[row]
-			? strings.format(checkboxTemplate, 'checked', this._options.title)
-			: strings.format(checkboxTemplate, '', this._options.title);
+			? strings.format(checkboxTemplate, 'checked', this.getAriaLabel(true))
+			: strings.format(checkboxTemplate, '', this.getAriaLabel(false));
 	}
 
 	checkboxTemplateCustom(row: number): string {
 		// use state after toggles
 		if (this._useState) {
 			return this._selectedCheckBoxLookup[row]
-				? strings.format(checkboxTemplate, 'checked', this._options.title)
-				: strings.format(checkboxTemplate, '', this._options.title);
+				? strings.format(checkboxTemplate, 'checked', this.getAriaLabel(true))
+				: strings.format(checkboxTemplate, '', this.getAriaLabel(false));
 		}
 
 		// use data for first time rendering
 		// note: make sure Init is called before using this._grid
 		let rowVal = (this._grid) ? this._grid.getDataItem(row) : null;
 		if (rowVal && this._options.title && rowVal[this._options.title] === true) {
 			this._selectedCheckBoxLookup[row] = true;
-			return strings.format(checkboxTemplate, 'checked', this._options.title);
+			return strings.format(checkboxTemplate, 'checked', this.getAriaLabel(true));
 		}
 		else {
 			delete this._selectedCheckBoxLookup[row];
-			return strings.format(checkboxTemplate, '', this._options.title);
+			return strings.format(checkboxTemplate, '', this.getAriaLabel(false));
 		}
 	}
 
 	private isCustomActionRequested(): boolean {
 		return (this._options.actionOnCheck === ActionOnCheck.customAction);
 	}
+
+	private getAriaLabel(checked: boolean): string {
+		return checked ? strings.format('"{0} {1}"', this._options.title, nls.localize("tableCheckboxCell.Checked", "checkbox checked")) :
```

Comment:
why not use a template string instead?

`strings.format('"{0} {1}"', this._options.title, nls.localize("tableCheckboxCell.Checked", "checkbox checked")) :`
becomes
`'"${this._options.title} ${nls.localize("tableCheckboxCell.Checked", "checkbox checked")}"'`